exercise_perseus_render/assets/src/vue/attemptprogress.vue

```diff
@@ -9,9 +9,9 @@
       </div>
     </div>
 
-    <p id="message" v-if="passRatioN === passNum">{{ $tr("get") }} <b>{{passRatioN}} {{ $tr("outof") }} {{passRatioM}}</b> {{ $tr("correct") }}</p>
-    <p id="message" v-if="passNum > 0 && passRatioN !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
-    <p id="message" v-if="passNum <= 0 && passRatioN !== passNum">{{ $tr("hooray") }}</p>
+    <p id="message" v-if="passRatioM === passNum">{{ $tr("get") }} <b>{{passRatioM}} {{ $tr("outof") }} {{passRatioN}}</b> {{ $tr("correct") }}</p>
+    <p id="message" v-if="passNum > 0 && passRatioM !== passNum">{{ $tr("get") }} <b>{{ passNum }}</b> {{ $tr("more") }}</p>
+    <p id="message" v-if="passNum <= 0 && passRatioM !== passNum">{{ $tr("hooray") }}</p>
```

Comment:
This wasn't part of this PR, but just a heads-up:

This method of combining numbers with translated strings won't work because it forces a certain order like

    ...english_text... {NUMBER} ...english_text... {NUMBER} ...english_text...

whereas in another language, it might be 

    {NUMBER} ... spanish_text ... {NUMBER}

In theory we should be using the [ICU Message Syntax](http://userguide.icu-project.org/formatparse/messages). However we don't yet have any working examples of this in use in the app. In the meantime, I'd recommend hard-coding these english strings and removing the `$tr` functionality because we don't want to set an incorrect precedent.